charts/rancher-vsphere-csi/values.yaml

```diff
@@ -145,37 +145,37 @@ global:
 # https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/3.0/vmware-vsphere-csp-getting-started/GUID-D4AAD99E-9128-40CE-B89C-AD451DA8379D.html#kubernetes-versions-compatible-with-vsphere-container-storage-plugin-1
 versionOverrides:
   # Versions from https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/release-3.0/manifests/vanilla/vsphere-csi-driver.yaml
```

Comment:
Want to update comment to point to `https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/release-3.1/manifests/vanilla/vsphere-csi-driver.yaml` I guess.

Even better, instead of referencing branch, reference tag: `https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/v3.1.2/manifests/vanilla/vsphere-csi-driver.yaml`